.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,63 @@
+name: NPM Publish
+
+on:
+  workflow_dispatch:
```

Comment:
Couldn't figure out from the jenkins file but I think this was a post-merge event? if we want to have it manual do you reckon we should add a line to readme?

Reply:
Ah yes, sorry this is a manual process now - I'll update the readme with why and how it is done

Reply:
Thanks